sm/enclave.c

```diff
@@ -241,7 +241,7 @@ static enclave_ret_code copy_word_to_host(uintptr_t dest_ptr, uintptr_t value)
 {
   enclave_ret_code ret = ENCLAVE_REGION_OVERLAPS;
 
-  int err = copy8_from_sm(dest_ptr, (uint64_t *)&value);
```

Comment:
Is this sized correctly on RV32?

Reply:
Yes, int-sizes are defined by XLEN, so this should be sized correctly.